.github/workflows/ci.yml

```diff
@@ -4,9 +4,9 @@ on:
     branches: [ "main" ]
   workflow_dispatch: {} # support manual runs
 
-# Prevent writing to the repository using the CI token.
 # Ref: https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#permissions
-permissions: read-all
+permissions:
+  packages: write
```

Comment:
Hope this is all we need.

Reply:
Based on the permissions page, I believe so. Let's give it a try.

https://docs.github.com/en/actions/security-guides/automatic-token-authentication#permissions-for-the-github_token